From 7d10dfa2f0d6a083c15f6c1eb97d9356e6afabfc Mon Sep 17 00:00:00 2001
From: Yang Wang <wang701@purdue.edu>
Date: Fri, 10 Feb 2017 14:11:45 -0500
Subject: [PATCH] Remove cpp compilation from Makefile

Signed-off-by: Yang Wang <wang701@purdue.edu>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 106c37c..690483e 100755
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-LIBSUBDIRS=	src src-cpp
+LIBSUBDIRS=	src
 
 CHECK_FILES+=	CONFIGURATION.md \
 		examples/rdkafka_example examples/rdkafka_performance \
-- 
2.7.4

